app/modules/site_settings/models.py

```diff
@@ -23,14 +23,13 @@ class SiteSetting(db.Model, Timestamp):
     __mapper_args__ = {
         'confirm_deleted_rows': False,
     }
-    EDM_PREFIX = 'site.'
 
     # public=False means the setting is only used in the backend
     HOUSTON_SETTINGS = {
         'email_service': {
             'type': str,
             'public': False,
-            'edm_definition': {
```

Comment:
anything that says edm_ is being removed, this is no longer the edm format, this is the format we will be using going forward.